amp-pwa-reader/gulpfile.js

```diff
@@ -63,7 +66,14 @@ function styles() {
     .pipe(gulp.dest(paths.styles.dest));
 }
 
-function scripts() {
+function cssJsScripts() {
```

Comment:
So, yes, I'm converting CSS to JS strings so that the web app can inject CSS into the shadow root without loading an external file (which proved to be unreliable).  Let me know if you think there's a prettier way of doing this.

Reply:
might be OK for now. I'd just rename the function to something like createCSSinJS or something a little more descriptive.

Reply:
sgtm!